sidechain/consensus/common/src/lib.rs

```diff
@@ -53,9 +53,9 @@ pub use error::*;
 use itp_types::parentchain::ParentchainCall;
 pub use peer_block_sync::*;
 
-pub trait Verifier<ParentchainBlock, SignedSidechainBlock>: Send + Sync
```

Comment:
all the sidechain stuff effectively only needs the header which is of equal type for all parentchains. Tried to untangle the mess....